### DISCUSSION TITLE
index.m3u8 file location

### DISCUSSION THREAD

**Discussion by LiamPerson** [bluenviron/mediamtx#719] [Questions]
Does anyone know where on a linux system this program puts the index.m3u8 file? 

e.g: `/root/rtsp/index.m3u8`

I am trying to set the document root of a url to the index.m3u8 file location.

**Comment by aler9** [comment]
Hello, the server doesn't save any file on disk, it keeps them into the RAM for performance reasons.

Nonetheless you can accomplish your goal by instantiating an HTTP server of your choice with any document root you want, and proxy requests to /stream/index.m3u8, /stream/123456.ts, etc, to the server, for instance this is a nginx configuration:

```
upstream rtspss {
    server 127.0.0.1:8888;
    keepalive 16;
}

server {
    listen 80;

    root /home/user/myfolder;

    location /stream {
        proxy_set_header Host $http_host;
        proxy_set_header X-Scheme $scheme;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection $connection_upgrade;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_pass http://rtspss;
    }
}
```

[a comment by an automated account is not shown]
